src/status_im/ui/components/colors.cljs

```diff
@@ -31,6 +31,7 @@
 
```

Comment:
are these approved by @denis-sharypin and @errorists ?

Reply:
yeah, what is this blue transparent 30, green transparent 10, are they new colours in the component lib @denis-sharypin ?

Reply:
@errorists  With blue transparent 30 it is my bad, I went wild.  @dmitryn could you replace it with `blue-light`, please?

Reply:
@flexsurfer green transparent 10, was added to our library after discussion with designers

Reply:
@denis-sharypin changed to `blue-light`